lib/datadog/core/remote/client.rb

```diff
@@ -117,7 +117,17 @@ def payload
             tracer_version: Core::Environment::Identity.tracer_version,
             service: Datadog.configuration.service,
             env: Datadog.configuration.env,
-            tags: [], # TODO: add nice tags!
+            tags: [
+              format('ruby.tracer.version:%s', Core::Environment::Identity.tracer_version),
+              format('ruby.runtime.platform:%s', RUBY_PLATFORM),
```

Comment:
Sorry, but I had to suggest? Any reason to not use string interpolation?
```suggestion
              "ruby.runtime.platform:#{RUBY_PLATFORM}",
```

Reply:
Ok, I actually have a non-nit-picking point here:
```ruby
# frozen_string_literal: true
require 'benchmark/ips'

Benchmark.ips do |x|
  x.time = 5
  x.warmup = 0.5

  x.report('format') { format('ruby.runtime.platform:%s', RUBY_PLATFORM) }
  x.report('#{}') { "ruby.runtime.platform:#{RUBY_PLATFORM}" }

  x.compare!
end

# Ruby 3.2.1
# Comparison:
#                  #{}:  8359588.7 i/s
#               format:  3939488.9 i/s - 2.12x  slower
```

Reply:
Very true!

Reply:
I'm going to archive that one to my [nascent "what's faster?" repo](https://github.com/lloeki/ruby-benchmarks) if you don't mind.

Reply:
Done